### PR TITLE
EBODAC-946: "DEV/QA - TASK - (Security) - Remove printing the passwor…

### PR DESCRIPTION
**Pull request by tomaszmueller**
…d when UI test is not passing"

### CODE REVIEW

**Comment by kkaczmarczyk** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

File: pom.xml [hunk 1]
```diff
@@ -381,7 +387,7 @@
                 <configuration>
                     <configLocation>${modules.root.dir}/checkstyle.xml</configLocation>
                     <suppressionsLocation>${modules.root.dir}/checkstyle-suppressions.xml</suppressionsLocation>
-                    <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                    <includeTestSourceDirectory>false</includeTestSourceDirectory>
```
Comment (pmuchowski) [inline pom.xml]: why you removed chackstyle from tests?

**Comment by tomaszmueller** [comment]
test this please

**Comment by kkaczmarczyk** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Pull request merged**